SConstruct

```diff
@@ -1681,10 +1681,12 @@ env['python_cmd_esc'] = quoted(env['python_cmd'])
 
 # Python Package Settings
 python_min_version = parse_version("3.8")
+ # next (unsupported) version
+python_next_version = parse_version("3.12")
```

Comment:
Would `python_max_version` be better for symmetry and to indicate the purpose of this number?

Reply:
Unfortunately, testing for versions <=3.11 fails, as it excludes patch numbers. Checking against the upcoming release is simpler.

Reply:
I still think that `python_next_version` isn't quite right... if we release 3.0 and it doesn't support 3.12, but October passes and 3.12 is released, this name implies to me it should be bumped to 3.13. I also get that it's not quite the max version, it's the max version + 1 🚲 🏠

Reply:
`python_max_p1_version` it is then (also updated the associated comment). I believe the 🚲🏡 looks nice at this point!